src/App.tsx

```diff
@@ -18,12 +18,14 @@ import { Settings } from "luxon";
 import { useMemo } from "react";
 import RestrictedContentProvider from "./components/providers/restricted-content-provider";
 import SprintViewScreen from "./components/screens/sprint-view-screen";
+import UserRoleUtils from "./utils/user-role-utils";
 
 /**
  * Application component
  */
 const App = () => {
   const language = useAtomValue(languageAtom);
+  const userIsDeveloper = UserRoleUtils.isDeveloper();
```

Comment:
Is this needed here , can't you just use the logic as its set up in the admin-screen and the home-screen and that's enough?